ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -25,5 +30,71 @@ public static bool UniverseContentsAreEqual(Universe obj1, Universe obj2)
             }
             return true;
         }
+
+        public static bool CellsAreEqual(Cell obj1, Cell obj2)
+        {
+            var isLocationSame = (obj1.Location.X == obj2.Location.X) && (obj1.Location.Y == obj2.Location.Y);
+            if (!isLocationSame)
+            {
+                return false;
+            }
+            if (obj1.CellState != obj2.CellState)
+            {
+                return false;
+            }
+            return true;

```

Comment:
I like the `isLocationSame` intermediate variable. We could do a similar thing for the cell state and then combine the return statements.
```suggestion
            var isLocationSame = (obj1.Location.X == obj2.Location.X) && (obj1.Location.Y == obj2.Location.Y);
            var isCellStateSame = obj1.CellState == obj2.CellState;
            return isLocationSame && isCellStateSame;
```